.github/workflows/integration_bugtracker.yml

```diff
@@ -48,6 +48,7 @@ jobs:
         export GH_BUGTRACKER_INTEGRATION_TEST_API_TOKEN="${{ secrets.GH_BUGTRACKER_INTEGRATION_TEST_API_TOKEN }}"
         export JIRA_BUGTRACKER_INTEGRATION_API_USERNAME="${{ secrets.JIRA_BUGTRACKER_INTEGRATION_API_USERNAME }}"
         export JIRA_BUGTRACKER_INTEGRATION_API_TOKEN="${{ secrets.JIRA_BUGTRACKER_INTEGRATION_API_TOKEN }}"
+        export AZURE_BOARDS_INTEGRATION_API_TOKEN="${{ secrets.AZURE_BOARDS_INTEGRATION_API_TOKEN }}"
```

Comment:
Missing Azure in the test matrix and the surrounding files. See `tests/jira/` for examples.